### PR TITLE
optional creatr trading pair inside create asset exchange

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: v3/storage/postgres/asset_exchange_create.go [hunk 1]
```diff
@@ -33,6 +33,22 @@ func (s *Storage) ConfirmCreateAssetExchange(id uint64) error {
 		if err != nil {
 			return err
 		}
+		for _, tradingPair := range r.TradingPairs {
```
Comment (secmask) [inline v3/storage/postgres/asset_exchange_create.go]: ConfirmCreateAssetExchange will be remove soon, setting-change API will be used as replacement

File: v3/http/asset_exchange_create.go [hunk 2]
```diff
@@ -53,5 +62,35 @@ func (s *Server) checkCreateAssetExchangeParams(createEntry common.CreateAssetEx
 	if asset.Transferable && common.IsZeroAddress(createEntry.DepositAddress) {
 		return common.ErrDepositAddressMissing
 	}
+	for _, tradingPair := range createEntry.TradingPairs {
```
Comment (secmask) [inline v3/http/asset_exchange_create.go]: beside of checking param, we will need to fill live info into trading pair

File: v3/http/asset_exchange_create.go [hunk 3]
```diff
@@ -24,6 +24,15 @@ func (s *Server) createCreateAssetExchange(c *gin.Context) {
 				httputil.WithField("asset_id", entry.AssetID), httputil.WithField("exchange_id", entry.ExchangeID))
 			return
 		}
+		for i, tp := range entry.TradingPairs {
```
Comment (secmask) [inline v3/http/asset_exchange_create.go]: please implement in setting-change method, this method will be remove soon.

**Comment by secmask** [comment]
@NNagato please fix the build error

File: v3/http/asset_exchange_create.go (hunk 2 above)
Comment (NNagato) [reply v3/http/asset_exchange_create.go]: changed as request

**Comment by NNagato** [comment]
> @NNagato please fix the build error

PR updated

**Pull request merged**